File: 백준/Bronze/32776. 가희와 4시간의 벽 2/가희와 4시간의 벽 2.cs
int S = int.Parse(Console.ReadLine());
string[] input = Console.ReadLine().Split();
int Ma = int.Parse(input[0]);
int F = int.Parse(input[1]);
int Mb = int.Parse(input[2]);

if (S <= 240 || S <= Ma + F + Mb)
    Console.WriteLine("high speed rail");
else
    Console.WriteLine("flight");

File: 백준/Bronze/32776. 가희와 4시간의 벽 2/가희와 4시간의 벽 2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int S = sc.nextInt();
        int Ma = sc.nextInt();
        int F = sc.nextInt();
        int Mb = sc.nextInt();

        if (S <= 240 || S <= Ma + F + Mb) {
            System.out.println("high speed rail");
        } else {
            System.out.println("flight");
        }

        sc.close();
    }
}
